nilearn/input_data/multi_nifti_masker.py

```diff
@@ -201,7 +200,9 @@ def fit(self, imgs=None, y=None):
         # Resampling: allows the user to change the affine, the shape or both.
         if self.verbose > 0:
             print("[%s.transform] Resampling mask" % self.__class__.__name__)
-        self.mask_img_ = self._cache(image.resample_img)(
+        # self.mask_img_ is memapped initially :  we ensure it stays that way
+        # when using cache
+        self.mask_img_ = self._cache(image.resample_img, mmap_mode='r')(
```

Comment:
The data array in the object is memmaped, which should be handled by joblib, right? I don't see the point in memmapping the Nifti1Image itself.